### ISSUE TITLE
proxy tests don't use tests/keys/mitmproxy-ca.pem

### ISSUE THREAD

**Issue by wRAR** [scrapy/scrapy#6258]
We pass `--certs tests/keys/mitmproxy-ca.pem`, but `--certs` is for specifying website certs, not CA certs, so it's ignored and the user-wide `~/.mitmproxy/mitmproxy-ca.pem` is used instead. We should either fix the way we specify the CA cert or (if that allows not using a CA cert at all) provide a website cert instead of a CA cert.

**Comment by Laerte** [comment]
Seems that we can use: https://docs.mitmproxy.org/stable/concepts-certificates/#using-a-custom-certificate-authority

**Comment by wRAR** [comment]
Yeah, that looks like the solution for the first option, though it notably takes a dir, not a file (but just passing `tests/keys` is unlikely to be problematic).
